docs/contents/users/install/kubeadm-onsite.md

```diff
@@ -0,0 +1,162 @@
+# Starting EKS-D with kubeadm (VM/bare metal)
+
+Installing and running an an Elastic Kubernetes Service Distro (EKS-D) on
+a VM or bare metal system requires setting up a few services and running
+kubeadm, as described below. 
+
+---
+**NOTE**:
+This procedure is provided for demonstration purposes and is not a supported product.
+The EKS version is EKS 1.18.  A separate product named [EKS-Anywhere](https://aws.amazon.com/eks/eks-anywhere/) 
+launches later in 2021 to let you automate your own bare metal and other EKS installations,
+---
+
+## Prerequisties
+
+Create a Linux system for the Kubernetes control plane and one or more nodes.
+This procedure was tested on an RPM-based Linux system. Start with the
+following prerequisites:
+
+1. Use a Linux system that kubeadm supports, as described in: [Before you begin](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin). Check that the system has the required amount of memory, CPU, and other resources. 
+
+1. Make sure SELinux is disabled by setting *SELINUX=disabled* in the */etc/syconfig/selinux* file. To turn it off immediately, type:
+
+    ```
```

Comment:
```suggestion
    ```bash
```

Reply:
Good suggestion. I'll do that.

Reply:
The ```bash seemed to work on the single-line code blocks. However, they broke the procedure numbering when used on the multi-line code blocks. I tried different indents, but that commands to all merge onto one line. I'm going to leave the text as it was since that seemed to work.